src/components/add-movie-form/add-movie-form.tsx

```diff
@@ -0,0 +1,99 @@
+import { Button } from 'components/button';
+import { Input } from 'components/input';
+import React, { FC, FormEvent, useState } from 'react';
+import {
+  AddMovieButtonContainer,
+  AddMovieFormContainer,
+  AddMovieFormInner,
+  AddMovieFormTitle,
+  AddMovieFormWrapper,
+  CloseButton,
+} from './styles';
+
+export const AddMovieForm: FC = () => {
+  const [value, setValue] = useState('');
+
+  const handleOnChange = (event: FormEvent<HTMLInputElement>): void => {
+    console.log('input: ', event.currentTarget.value);
+
+    setValue(event.currentTarget.value);
+  };
+
+  return (
+    <AddMovieFormWrapper>
+      <AddMovieFormContainer>
+        <CloseButton />
+        <AddMovieFormTitle>Add Movie</AddMovieFormTitle>
+        <AddMovieFormInner>
+          <Input
+            topic
+            label="Title"
+            name="title"
+            type="text"
+            placeholder="Moana"
+            onChange={handleOnChange}
+            value={value}
+          />
+          <Input
+            release
+            label="Release date"
+            name="release"
+            type="text"
+            placeholder="Select Date"
+            onChange={handleOnChange}
+            value={value}
+          />
+          <Input
+            movie
+            label="Movie url"
+            name="movie"
+            type="text"
+            placeholder="Movie url here"
+            onChange={handleOnChange}
+            value={value}
+          />
+          <Input
+            movie
+            label="Rating"
+            name="rating"
+            type="text"
+            placeholder="7.8"
+            onChange={handleOnChange}
+            value={value}
+          />
+          <Input
+            genre
+            label="Genre"
+            name="genre"
+            type="text"
+            placeholder="Select Genre"
+            onChange={handleOnChange}
+            value={value}
+          />
+          <Input
+            runtime
+            label="Runtime"
+            name="runtime"
+            type="text"
+            placeholder="Runtime here"
+            onChange={handleOnChange}
+            value={value}
+          />
+          <Input
+            overview
+            label="Overview"
+            name="overview"
+            type="text"
+            placeholder="Overview here"
+            onChange={handleOnChange}
+            value={value}
+          />
+        </AddMovieFormInner>
+        <AddMovieButtonContainer>
+          <Button reset type="reset" onClick={null} text="Reset"></Button>
```

Comment:
better to make on click optional since the button could be also used with type submit and then onClick event usually does not make sense

Reply:
yep, it's just a scaffold component. In next HW I'll add functionality.